tools/ci_build/github/azure-pipelines/binary-size-checks-pipeline.yml

```diff
@@ -23,6 +23,17 @@ jobs:
     clean: all
   pool: Linux-CPU-2019
 
+  strategy:
+    matrix:
+      AndroidMinimalBaseline:
+        ConfigId: minimal-baseline
+        BuildConfigFile: "/onnxruntime_src/tools/ci_build/github/linux/ort_minimal/build_check_binsize_config/android_minimal_baseline.config"
+        BinarySizeThresholdInBytes: 1306224
+      AndroidMinimalBasicWithSomeOps:
+        ConfigId: android-minimal-basic-with-some-ops
+        BuildConfigFile: "/onnxruntime_src/tools/ci_build/github/linux/ort_minimal/build_check_binsize_config/android_minimal_basic_with_some_ops.config"
```

Comment:
nit: if this is the mobile package set of ops can we call it ...-with-mobile-package-ops?

What does the 'basic' in the name represent?

Reply:
mobile-package-ops sounds good. I was struggling with the name a bit. "basic" is an attempt to convey that certain features (e.g. sparse tensors) are disabled.

Reply:
updated it to be a variation of minimal-baseline with the mobile package ops included. we can add a different configurations that we are interested in later